shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -68,6 +68,58 @@ - (instancetype)initWithConnection:(NSNumber*)connection
 
 #pragma mark -
 
+// The information of a view that has been attached to the engine.
+@interface FlutterEngineViewRecord : NSObject
+- (instancetype)initWithController:(nonnull FlutterViewController*)controller
+                            engine:(nonnull FlutterEngine*)engine
+                  semanticsEnabled:(BOOL)semanticsEnabled;
+
+- (BOOL)semanticsEnabled;
+
+- (void)setSemanticsEnabled:(BOOL)semanticsEnabled;
+
+@property(nonatomic, readonly, weak) FlutterViewController* controller;
+
+@property(nonatomic, readonly) std::shared_ptr<flutter::AccessibilityBridgeMac> bridge;
```

Comment:
Can this be `std::unique_ptr`?

Reply:
This is a `shared_ptr` because other classes (`FlutterPlatformNodeDelegate`) need to get a `weak_ptr` to use it. But I think we might be able to change to a reference, since we're asserting that the bridge is non-null anyway. I'll experiment and report the result.

Reply:
No, we can't. It turns out that `FlutterPlatformNodeDelegateMac` takes a `weak_ptr` as an argument and uses the engine in the future.